var/spack/repos/builtin/packages/geomodel/package.py

```diff
@@ -72,7 +72,7 @@ class Geomodel(CMakePackage):
 
     def cmake_args(self):
         def cmake_variant(cmake_label, spack_variant):
-            enabled = spec.satisfies("+" + spack_variant)
+            enabled = self.spec.satisfies("+" + spack_variant)
```

Comment:
Can we use `self.define_from_variant` as in other packages?

Reply:
Updated as requested.